storage/integration_test.go

```diff
@@ -524,96 +524,87 @@ func TestIntegration_BucketLifecycle(t *testing.T) {
 }
 
 func TestIntegration_BucketUpdate(t *testing.T) {
-	ctx := context.Background()
-	client := testConfig(ctx, t)
-	defer client.Close()
-	h := testHelper{t}
-
-	b := client.Bucket(uidSpace.New())
-	h.mustCreate(b, testutil.ProjID(), nil)
-	defer h.mustDeleteBucket(b)
+	ctx := skipJSONReads(context.Background(), "no reads in test")
+	multiTransportTest(ctx, t, func(t *testing.T, ctx context.Context, _ string, prefix string, client *Client) {
+		h := testHelper{t}
 
-	attrs := h.mustBucketAttrs(b)
-	if attrs.VersioningEnabled {
-		t.Fatal("bucket should not have versioning by default")
-	}
-	if len(attrs.Labels) > 0 {
-		t.Fatal("bucket should not have labels initially")
-	}
+		b := client.Bucket(prefix + uidSpace.New())
+		h.mustCreate(b, testutil.ProjID(), nil)
+		defer h.mustDeleteBucket(b)
 
-	// Using empty BucketAttrsToUpdate should be a no-nop.
```

Comment:
It looks like you removed this check with an empty `BucketAttrsToUpdate`, any particular reason?

Reply:
Yes, I removed it because gRPC returns an error in that case, while it is a no-op for HTTP. Is there value in keeping this test, do you think? If there is, we'd have to either accept both cases here (could make for some not-nice looking code), or separate this check into 2 tests, one for HTTP and one for JSON

Reply:
Discussed offline, opened #7685 to address the discrepancy issue.